client/pages/sections/protocols/sections.js

```diff
@@ -139,7 +148,9 @@ const mapStateToProps = ({
   // show all sections for legacy
   sections: isGranted && schemaVersion !== 0
     ? pickBy(sections, section => section.granted)
-    : sections
+    : sections,
+  latest,
+  granted
```

Comment:
this file no longer cares about latest/granted - why don't we connect them in the new component you made?